sdc/tests/test_sdc_numpy.py

```diff
@@ -188,6 +188,57 @@ def sdc_impl(a):
             with self.subTest(data=case):
                 np.testing.assert_array_equal(sdc_func(a), ref_impl(a))
 
+    def test_copy(self):
+        def ref_impl(a):
+            return np.copy(a)
+
+        def sdc_impl(a):
+            return numpy_like.copy(a)
```

Comment:
If you want you could do something like that [sdc/tests/test_dataframe.py#L1240-L1247](https://github.com/IntelPython/sdc/blob/master/sdc/tests/test_dataframe.py#L1240-L1247).